pkg/operator/sync.go

```diff
@@ -113,3 +103,130 @@ func (optr *Operator) waitForDeploymentRollout(resource *appsv1.Deployment) erro
 		return false, nil
 	})
 }
+
+func newDeployment(config *OperatorConfig, features map[string]bool) *appsv1.Deployment {
+	replicas := int32(1)
+	template := newPodTemplateSpec(config, features)
+
+	return &appsv1.Deployment{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "machine-api-controllers",
+			Namespace: config.TargetNamespace,
+			Labels: map[string]string{
+				"api":     "clusterapi",
+				"k8s-app": "controller",
+			},
+		},
+		Spec: appsv1.DeploymentSpec{
```

Comment:
missing 
```yaml
securityContext:
    runAsNonRoot: true
    runAsUser: 65534
```

Reply:
it exists under the `PodTemplate` https://github.com/openshift/machine-api-operator/pull/280/files#diff-fa45321336db7ad1cedc28bf643a4f97R170